### PR TITLE
Anonymizer - Pass bytes key to Encrypt / Decrypt

### PR DESCRIPTION
**Pull request by SharonHart**
## Change Description

In addition to string typed key, add the ability to pass a bytes key (for as-a-package use)
## Issue reference

This PR fixes issue #1131 

## Checklist

- [x] I have reviewed the [contribution guidelines](https://github.com/microsoft/presidio/blob/main/CONTRIBUTING.md)
- [x] I have signed the CLA (if required)
- [x] My code includes unit tests
- [x] All unit tests and lint checks pass locally
- [x] My PR contains documentation updates / additions if required

### CODE REVIEW

File: presidio-anonymizer/tests/integration/test_deanonymize_engine.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def test_given_operator_decrypt_with_valid_params_then_decrypt_text_successfully
     decryption = engine.deanonymize(
         text,
         encryption_results,
-        {"DEFAULT": OperatorConfig(Decrypt.NAME, {"key": "WmZq4t7w!z%C&F)J"})},
+        {"DEFAULT": OperatorConfig(Decrypt.NAME, {"key": b'WmZq4t7w!z%C&F)J'})},
```
Comment (niwilso) [inline presidio-anonymizer/tests/integration/test_deanonymize_engine.py:23]: **nit**: Should the single quotes be double quotes to be consistent with other quotes throughout the file?

**Review by niwilso — APPROVED** [review]

File: presidio-anonymizer/tests/integration/test_deanonymize_engine.py (hunk 1 above)
Comment (SharonHart) [reply presidio-anonymizer/tests/integration/test_deanonymize_engine.py:23]: maybe, I am used to seeing byte string represented with single quotes, not sure what is the convention there, but all bytestrings in the repo are with single quotes so it is consistent

Comment (omri374) [inline presidio-anonymizer/tests/integration/test_deanonymize_engine.py:23]: Does this require a change [in this notebook](https://github.com/microsoft/presidio/blob/main/docs/samples/python/encrypt_decrypt.ipynb) and in the [tutorial](https://github.com/microsoft/presidio/blob/main/docs/tutorial/12_encryption.md]?

File: presidio-anonymizer/tests/services/test_aes_cypher.py [hunk 2]
```diff
@@ -14,6 +16,7 @@
         (b'1111111111111111', "面汤"),  # Chinese text
```
Comment (omri374) [inline presidio-anonymizer/tests/services/test_aes_cypher.py:16]: If a string is a valid input which would get encoded as part of the process, should we have tests checking the validity of a string input as well?

File: presidio-anonymizer/tests/integration/test_deanonymize_engine.py (hunk 1 above)
Comment (SharonHart) [reply presidio-anonymizer/tests/integration/test_deanonymize_engine.py:23]: No, its another option, kept one test with that option, one with the string option

File: presidio-anonymizer/tests/services/test_aes_cypher.py (hunk 2 above)
Comment (SharonHart) [reply presidio-anonymizer/tests/services/test_aes_cypher.py:16]: At this level only bytes are passed, added both string and bytes option in tests for upper level

**Review by omri374 — APPROVED** [review]

**Pull request merged**